packages/components/src/Page/Page.tsx

```diff
@@ -186,3 +193,10 @@ export function Page({
     </div>
   );
 }
+
+export const getActionProps = (actionProps: ButtonActionProps): ButtonProps => {
+  const buttonProps = { ...actionProps };
+  if (actionProps.ref) delete buttonProps.ref;
+
+  return buttonProps;
+};
```

Comment:
This takes in the primary or secondary action prop, and cleans it by removing the ref prop so it is not passed into the button component.